tooling/eslint-config/eslint.config.mjs

```diff
@@ -9,4 +9,9 @@ export default [
   ...tseslint.configs.recommended,
   eslintConfigPrettier,
   { ignores: ['**/node_modules', '**/dist'] },
+  {
+    rules: {
+      '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
```

Comment:
I missed this point but did you add that to be able to use `_` for unused parameters in callbacks? Like ...
```js
someArray.map((_, index) => index + 1)
```

Reply:
Yes, and also to be able to give semantic names to pairs, for example, `([_key, value]) => value` instead of `(pair) => pair[1]`